docs/self-sufficient-bundle.md

```diff
@@ -12,9 +12,9 @@ services to do their work.
 |    `crc-pullsecret.service`    |               both               |         /opt/crc/pull-secret         |         none         |
 |      `crc-dnsmasq.service`     |               both               |                 none                 |         none         |
 | `crc-routes-controller.service`|               both               |                 none                 |         none         |
-|    `ocp-cluster-ca.service`    |               ocp                |        /opt/crc/custom-ca.crt        |     CRC_NG=1         |
+|    `ocp-cluster-ca.service`    |               ocp                |        /opt/crc/custom-ca.crt        |         none         |
```

Comment:
Not sure why this is changed in this commit? Does this depend on ocp-custom-domain.service?

Reply:
looks like it was by mistake present and `ocp-custom-domain` already had this check about cloud-ng env.